webapp/lib/controller.dart

```diff
@@ -169,59 +131,146 @@ void loadDashboardView() {
 
 void loadBatchRepliesConfigurationView() {
   view.contentView.renderView(new view.BatchRepliesConfigurationView(model.changeCommsPackage));
-  command(UIAction.loadBatchRepliesPackageConfiguration, null);
 }
 
 void loadEscalatesConfigurationView() {
   view.contentView.renderView(new view.EscalatesConfigurationView());
-  command(UIAction.loadEscalatesPackageConfiguration, null);
 }
 
 loadProjectConfigurationView() {
   view.contentView.renderView(new view.ProjectConfigurationView());
 }
 
-void fetchConfigurationData() {
-  configurationTagData = model.configurationData;
-  additionalConfigurationTags = model.configurationTags;
-  configurationResponseLanguages = model.configurationReponseLanguageData;
+// Tag Operations
+enum TagOperation {
+  ADD,
+  UPDATE,
+  REMOVE
+}
+
+void _addTag(String tag, model.TagStyle tagStyle, Map<String, model.TagStyle> tagType, [bool isEditable = false]) {
```

Comment:
nit: rename `tagType` to `tagCollection` (both here and in the other functions below)

Reply:
Renamed